### PR TITLE
refactor style

### PR DESCRIPTION
**Pull request by KamiAmiga**
Refactor pack less to follow new guideline

### CODE REVIEW

File: client/app/telecom/pack/slots/voipLine/activation/pack-voipLine-activation.html [hunk 1]
```diff
@@ -80,11 +80,12 @@ <h2 class="widget-presentation-title" data-translate="telefony_activation_widget
                                 <span data-translate="telefony_activation_no_more_port_modem"></span>
                             </div>
                         </div>
-                        <div data-ng-if="hardware.line.needHardware">
+                        <div data-ng-if="hardware.line.needHardware" class="top-space-m8">
                             <!-- choices -->
                             <div data-uib-carousel id="{{'carousel' + hardware.index}}"
                                  data-ng-if="!hardware.line.hardware"
-                                 active="hardware.carouselIndex">
+                                 active="hardware.carouselIndex"
```
Comment (antleblanc) [inline client/app/telecom/pack/slots/voipLine/activation/pack-voipLine-activation.html]: `active` => `data-active`

File: client/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js [hunk 2]
```diff
@@ -1,171 +1,169 @@
...
-            break;
-        case "cancelled":
-        case "close":
-        case "deleting":
-        case "slamming":
-            self.statusLabel = "<i class=\"ovh-font ovh-font-failure text-danger fs16 right-space-m8\"></i> " + $translate.instant("xdsl_details_status_" + status);
-            break;
-        default :
-            self.statusLabel = status;
+angular.module("managerApp").controller("XdslAccessCtrl",
```
Comment (antleblanc) [inline client/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js]: Do you have any reason of modifying this file? I think that `grunt eslint` will warn you now!

File: client/app/telecom/pack/xdsl/access/statistics/pack-xdsl-statistics.html [hunk 3]
```diff
@@ -1,220 +1,197 @@
...
-        <div class="row top-space-m32">
-            <div class="col-md-12">
+        </form>
 
-                <div class="space-m20 alert alert-warning " role="alert"
-                    data-ng-if="!XdslStatistics.synchronization.haveSeries && !XdslStatistics.synchronization.loading"
-                    data-translate="xdsl_statistics_error">
-                    <i class="ovh-font ovh-font-filled-warning"></i>
-                </div>
+        <highchart id="synchronizationChart"
```
Comment (antleblanc) [inline client/app/telecom/pack/xdsl/access/statistics/pack-xdsl-statistics.html]: Hmm caution with this file! There are no more `<highchart>`!

File: client/app/telecom/pack/slots/informations/pack-informations.html [hunk 4]
```diff
@@ -85,16 +85,17 @@
...
 
         <!-- RESILIATION DATE -->
-        <div data-ng-if="PackInformation.resiliationFollowUp" class="row info-row info-row-darker">
+        <div data-ng-if="PackInformation.resiliationFollowUp" class="row info-row ">
             <div class="col-sm-4" data-translate="pack_xdsl_resiliation_date"></div>
             <div class="col-sm-8 description">
-                <a data-ui-sref="telecom.pack.resiliation"
+                <a class="service-button"
+                   data-ui-sref="telecom.pack.resiliation"
                    title="{{ 'common_resiliation_cancel' | translate }}">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/informations/pack-informations.html]: better : `data-translate-attr="{ title: 'common_resiliation_cancel' }"`

File: client/app/telecom/pack/slots/voipLine/activation/pack-voipLine-activation.html [hunk 5]
```diff
@@ -198,27 +199,29 @@ <h2 class="widget-presentation-title" data-translate="telefony_activation_shippi
                 <div class="row animate-addresses" data-ng-show="PackVoipLineActivationCtrl.shippingMode === 'transporter'">
                     <div class="top-space-m16" data-ng-repeat="addressGroup in PackVoipLineActivationCtrl.framedShippingAddresses">
                         <div class="col-md-4" data-ng-repeat="address in addressGroup track by address.shippingId">
```
Comment (Jisay) [inline client/app/telecom/pack/slots/voipLine/activation/pack-voipLine-activation.html]: direct parent of `col-*-*` must be a `row`

**Pull request merged**